integration/aggregate_compat_test.go

```diff
@@ -365,6 +370,12 @@ func TestAggregateCompatSort(t *testing.T) {
 			pipeline:   bson.A{bson.D{{"$sort", bson.D{}}}},
 			resultType: emptyResult,
 		},
+		"Location15998": { // aka DotNotationMissingField
```

Comment:
In this test, test cases are called after error codes, so I keep the same naming. However, in the other places, such test was called `DotNotationMissingField`, so for this one I call it `Location15998`, but in the other places I call it `DotNotationMissingField` - as before. And I added a comment to be able to find this test case when search by "DotNotationMissingField".

Reply:
Is there a way to replace those test case names with meaningful names? 
Location15976 – no keys provided
Location15975 – invalid sort order value provided
Location15973 – invalid sort order value type

Reply:
@chilagrow what do you think? To me, the naming `LocationXYZ` makes sense as it's easy to understand which cases are covered by it, but on the other hand, this is not how such tests are named in the other parts of code. As you are the owner of this naming strategy, I need your permission to rename it to what Dmitry suggests :)

Reply:
So I started naming them `Location..` because I wanted to have short and easy to map test names to the error it was expecting to receive. But it's not so easy to tell apart. Of the top of my head, meaningful is something like this? Anyway agree with @w84thesun on having something more meaningful :hugs: 

Location15976 – no keys provided produces `ErrSortMissingKey` -> `SortMissingKey`
Location15975 – invalid sort order value provided produces `ErrSortBadOrder` -> `SortBadOrder`
Location15973 – invalid sort order value type produces `ErrSortBadExpression` -> `SortBadExpression`

Also if the test case name is too long, tests fail because test name is used for the collection name :see_no_evil:.

Reply:
Fixed in all `integration`, so the naming is consistent